src/ui/app/tabs/hw.jsx

```diff
@@ -42,10 +42,11 @@ import { MdUsb } from 'react-icons/md';
 import { Planet } from 'react-kawaii';
 import { useCaptureEvent } from '../../../features/analytics/hooks';
 import { Events } from '../../../features/analytics/events';
+import { ledgerUSBVendorId } from '@ledgerhq/devices';
 
-const MANUFACTURER = {
-  ledger: 'Ledger',
-  trezor: 'SatoshiLabs',
+const VENDOR_IDS = {
+  ledger: [ledgerUSBVendorId],
+  trezor: [0x534c, 0x1209], // Model T HID 0x534c and others 0x1209
```

Comment:
Is Model T the only exception here?

Reply:
As far as I can tell looking at this:

https://github.com/vacuumlabs/trezor-suite/blob/66aa7e8486a980f18ab360ebce4aeee48ce0ac98/packages/transport/src/constants.ts#L11-L19

yes

Reply:
Done (and squashed)